arangod/Cache/Table.cpp

```diff
@@ -43,6 +43,15 @@ void Table::GenericBucket::unlock() {
   _state.unlock();
 }
 
+void Table::GenericBucket::clear() {
+  _state.lock(UINT64_MAX, [this]() -> void {
+    _state.clear();
+    memset(this + sizeof(BucketState), 0,
+           sizeof(GenericBucket) - sizeof(BucketState));
```

Comment:
Same problem as above - this would result in a write outside the bounds of the bucket instance.